styles.css

```diff
@@ -112,15 +113,51 @@ body {
   opacity: 1;
 }
 
-.info {
-  display: flex;
-  justify-content: space-between;
+.main-content {
+  padding: 0 63% 200px 0;
+  background: top right no-repeat url(img/image.svg);
+  margin-bottom: 133px;
+}
+
+.main-content__title {
+  margin: 0 0 40px 0;
+  font: bold 66px/87px 'PT Serif', Arial, Helvetica, sans-serif;
+}
+
+.main-content__title_highlight {
+  background: #ECF3FF;
+}
+
+.main-content__description {
+  margin: 0 0 41px 0;
+  font: 400 22px/26px 'Roboto', Arial, Helvetica, sans-serif;
+}
+
+.main-content__button {
+  display: inline-block;
+  color: #fff;
+  font: 500 18px/20px 'Roboto', Arial, Helvetica, sans-serif;
+  padding: 14px 16px;
+  background: #0056EC;
```

Comment:
Codacy found an issue: [Expected "#0056EC" to be "#0056ec" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/kastykina_app.creation/pullRequest?prid=6278926)